docs/get-started/v2/get-started/prebuilt/prebuilt.mdx

```diff
@@ -57,4 +76,153 @@ Once you have a Prebuilt room link ready, you can share it with your users on th
+
+
+<StepsContainer id="implement-breakout-room">
+
+### Create and Get room code
+HMSPrebuilt uses [room code](/concepts/v2/concepts/rooms/room-codes/room-codes) to get initialised, it allow users to uniquely join a `room_id` with a specifc `role`. Head over to 
+- [Create Room Codes](/docs/get-started/v2/get-started/prebuilt/room-codes/overview#creating-room-codes)
+- [Get Room Code](/docs/server-side/v2/api-reference/room-codes/create-room-code-api)
+
+
+### Embedding HMSPrebuilt component
+    - Import - HMSPrebuilt component requires importing HMSRoomKit 
+    - Initialise - Along with room code, you can additionally add options such as `userName` and `userID` in the component
+        - `userName` - prefills the user name on HMSPrebuilt preview screen
+        - `userID` - use your user IDs to track joins, if not passed 100ms auto-generates a userID
+    - Customise - You can override themes and colors to match it with your app themes
+
+Sample code snippet to embed Prebuilt component. Head over for a step by step guides for [Flutter](docs/flutter/v2/quickstart/prebuilt-flutter), [React Native](/docs/react-native/v2/quickstart/quickstart), [Android](/docs/android/v2/quickstart/prebuilt-android), [iOS](docs/iOS/v2/quickstart/prebuilt-iOS)
+
+<Tabs id="sample-code" items={['React Native','Android','iOS','Flutter']} />
+
+<Tab id='sample-code-0'>
+
+```js section=SampleCode tab=React Native
+// Simple Client Usage Example
+const MeetingScreen = () => {
+    return (
+        <HMSPrebuilt
+            roomCode=""
+            options={{
+                userName: "Jatin Nagar",
+                userId: "randomstring"
+            }}
+        />
+    );
+};
+
+// Example App Usage Example
+export const HMSPrebuiltScreen = () => {
+  const route = useRoute<RouteProp<AppStackParamList, 'HMSPrebuiltScreen'>>();
+
+    // `route.params` should have all the config options
+    // required for `HMSPrebuilt` component
+
+  const roomCode = route.params?.roomCode;
+
+  if (!roomCode) {
+    return <Text>Room Code is Required</Text>;
+  }
+
+  return <HMSPrebuilt roomCode={roomCode} options={route.params} />;
+};
+```
+</Tab>
+
+<Tab id='sample-code-1'>
+```kotlin section=SampleCode sectionIndex=2 tab=Kotlin
+
+// Add HMSRoomKit dependency in your gradle file
+implementation "live.100ms:room-kit:$roomKitVersion"
+
+//Add package import in your activity or fragment
+import live.hms.roomkit.ui.HMSRoomKit
+
+//To launch prebuilt from your activity or fragment
+HMSRoomKit.launchPrebuilt(roomCode: String, context: ActivityContext, options: HMSPrebuiltOptions? = null)
+
+class HMSPrebuiltOptions {
+    val userName : String?, //Peer user name
+    val userId: String?, // Optional Customer given User-id 
+    val endPoints: Map?, // Optional end points. Not needed for production apps
+  val debugInfo: Bool = false, // Optional debug info. Not needed for production apps
+}
+
+
+```
+</Tab>
+
+<Tab id='sample-code-2'>
+
+```swift section=SampleCode  tab=iOS
```

Comment:
lets have coming soon tab for iOS or lets not mention at all for iOS for first release